core/web/evm_forwarders_controller.go

```diff
@@ -79,6 +92,13 @@ func (cc *EVMForwardersController) Delete(c *gin.Context) {
 		jsonAPIError(c, http.StatusUnprocessableEntity, err)
 		return
 	}
+	// this is a breaking change, but it's necessary to support multiple chains
+	chainId, err := stringutils.ToInt64(c.Param("chainID"))
```

Comment:
I don't know much about the forwarder, but if we didn't support multiple chains before... how did it work, you could only use forwarders on the default chain or something?

Reply:
no, we were leaking the DB id and expecting user to use that. that DB id was unique b/c all the forwarders were in one table